skyportal/tests/tools/test_sky.py

```diff
@@ -0,0 +1,26 @@
+import numpy as np
+from astropy.time import Time
+
+
+def test_airmass(public_source, ztf_camera):
+    telescope = ztf_camera.telescope
+    times = Time(np.linspace(58000, 59000), format='mjd')
+    airmass_calc = public_source.airmass(telescope, times).value
+    airmass_true = np.asarray([ -6.98036718, -23.34945066,   4.33830158,
```

Comment:
What does negative mean? We should have some `below_the_horizon` value.

Reply:
Negative means below the horizon. For below the horizon, how about `np.inf`?

Reply:
Implemented as np.inf in ec23e7d